index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <div class="center">
+        <h2>freeCodeCamp Survey Form</h2>
+        <p>Thank you for taking the time to help us improve the platform</p>
+    </div>
```

Comment:
Please consider using the `header` tag for this section of the code as it contains the header section of the project. HTML5 semantics are preferred to `div`

Reply:
> * [ ]  Nice job asking for changes in a friendly and polite way but please make the reason why the reviewee had to this change more clear like "It would be better semantically if you use ............."

Please consider using the `header` tag for this section of the code as it contains the header section of the project. it would be better semantically if you use `header` instead of `div`.